platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/EventImplTests.java

```diff
@@ -38,29 +38,27 @@
 import java.util.List;
 import java.util.Random;
 import java.util.Set;
-import java.util.stream.Stream;
+import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
-import org.junit.jupiter.params.provider.MethodSource;
 
 public class EventImplTests {
-    private final TestingEventBuilder testingEventBuilder = TestingEventBuilder.builder();
+    private Random random;
+    private TestingEventBuilder testingEventBuilder;
 
-    private static Stream<Arguments> params() {
-        return Stream.of(
-                // This seed will cause the testFindSystemTransactions() test to fail if the set is not ordered
-                Arguments.of(2470662876573509733L, 10_000), Arguments.of(null, 10_000));
+    @BeforeEach
+    void setUp() {
+        random = RandomUtils.getRandomPrintSeed();
+        testingEventBuilder = TestingEventBuilder.builder(random);
     }
 
     @Test
     @DisplayName("findSystemTransactions() no system transactions")
     void testNoSystemTransaction() {
         final SwirldTransaction[] transactions = TransactionUtils.incrementingSwirldTransactions(100);
 
-        final EventImpl event =
```

Comment:
why not use buildEventImpl()?

Reply:
It's not necessary to do anything special if the event has no parents. But I can push a change to use the new `EventImpl` utility method, for the sake of consistency

Reply:
we discussed this on zoom